src/plugins/intel_npu/src/al/include/config/common.hpp

```diff
@@ -4,7 +4,7 @@
 
 #pragma once
 
-#include "intel_npu/al/config/config.hpp"
+#include "config/config.hpp"
```

Comment:
We should keep the full include path. That was intentional.
This is cross repo header, so short include path like config/config.hpp is prone to clashes.
Not sure what is the OV convention, but if we use cmake target for propagating the path, we need that "double" include dir structure, so it's clear what's being included and avoid name clashes.

Reply:
done, I changed it everywhere. Just to be very clear.